projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -11,50 +11,49 @@ const getNotices = ( tierMaximumRecords = null ) => {
 	return {
 		1: {
 			id: 1,
+			header: __( 'Search was unable to index your content', 'jetpack-search-pkg' ),
 			message: __(
-				"Search was unable to locate your content. Jetpack's servers ran into a problem when trying to communicate with your site, which is needed for Search to work properly.",
+				"Jetpack's servers ran into a problem when trying to communicate with your site.",
 				'jetpack-search-pkg'
 			),
 			isImportant: true,
 		},
 		2: {
 			id: 2,
-			message: __( 'Your content has not yet been indexed for Search', 'jetpack-search-pkg' ),
+			header: __( 'Your content has not yet been indexed for Search', 'jetpack-search-pkg' ),
+			message: __(
+				'If you have recently set up Search, please allow a little time for indexing to complete.',
+				'jetpack-search-pkg'
+			),
 		},
 		3: {
 			id: 3,
+			header: __(
+				"We weren't able to locate any content for Search to index",
+				'jetpack-search-pkg'
+			),
 			message: __(
-				"We weren't able to locate any content for Search to index. Perhaps you don't yet have any posts or pages?",
+				"This can happen if you don't have any posts or pages yet.",
 				'jetpack-search-pkg'
 			),
 		},
 		4: {
-			id: 4,
-			message: sprintf(
-				// translators: %s: site's current plan record limit
-				__(
-					'You recently surpassed %d records and will be automatically upgraded to the next billing tier <p> learn more <p>',
-					'jetpack-search-pkg'
-				),
-				tierMaximumRecords
-			),
-			link: {
-				text: __( 'learn more', 'jetpack-search-pkg' ),
-				url: 'https://jetpack.com/support/search/product-pricing/',
-			},
-		},
-		5: {
 			id: 5,
```

Comment:
should the id be '4'?

Reply:
Indeed it should. Good spot 👍 

I suspect the `id` key isn't actually used.